.vscode/launch.json

```diff
@@ -10,21 +10,31 @@
 			"type": "extensionHost",
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
-			"args": ["--extensionDevelopmentPath=${workspaceFolder}" ],
+			"args": [
+				"--extensionDevelopmentPath=${workspaceFolder}"
+			],
 			"stopOnEntry": false,
 			"sourceMaps": true,
-			"outFiles": ["${workspaceFolder}/dist/*.js"],
+			"outFiles": [
+				"${workspaceFolder}/dist/*.js"
+			],
 			"preLaunchTask": "npm: webpack-dev"
 		},
 		{
 			"name": "Launch Tests",
 			"type": "extensionHost",
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
-			"args": ["${workspaceFolder}/test/resources/sampleNodeApp/", "--disable-extensions", "--extensionDevelopmentPath=${workspaceRoot}", "--extensionTestsPath=${workspaceRoot}/out/test" ],
+			"args": [
```

Comment:
Why --disable-extensions is removed? Anyways our testing should independent of vscode-commons. Its absence shouldn't make test failures.

Reply:
as we have added in the extentionDependency test case try to check if vscode-commons is present.